samples/image-metadata/src/components/AppView/AppView.js

```diff
@@ -0,0 +1,156 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import camelCase from 'lodash.camelcase';
+import { Heading, Typography, Paragraph } from '@contentful/forma-36-react-components';
+
+import { ConfigurationContent } from './ConfigurationContent';
+import { InstallationContent } from './InstallationContent';
+import { Divider } from '../Divider';
+import { styles } from './styles';
+
+export class AppView extends Component {
+  static propTypes = {
+    sdk: PropTypes.object.isRequired
+  };
+
+  state = {
+    allContentTypesIds: [],
+    appIsInstalled: null,
+    contentTypeId: camelCase('Image Wrapper'),
+    contentTypeName: 'Image Wrapper'
+  };
+
+  async componentDidMount() {
+    const { sdk } = this.props;
+    const { app } = sdk.platformAlpha;
+
+    const appIsInstalled = await app.isInstalled();
+
+    const allContentTypes = await sdk.space.getContentTypes();
+    const allContentTypesIds = allContentTypes.items.map(({ sys: { id } }) => id);
+
+    // Following eslint error is caused due to using async/await
+    // eslint-disable-next-line react/no-did-mount-set-state
+    this.setState({ appIsInstalled, allContentTypesIds });
+
+    if (!appIsInstalled) {
```

Comment:
Remove this `if` statement but keep the inner code. You want to give `onConfigure` a callback regardless of if the app is installed or not. This is because when I hit save (after the app is installed), it will still call this method.

Reply:
In addition to that I'd also call `onConfigure` as soon as possible, not after async actions